cli/module_registry.go

```diff
@@ -72,7 +73,7 @@ func CreateModuleCommand(c *cli.Context) error {
 	// Check to make sure the user doesn't accidentally overwrite a module manifest
 	manifestFilepath := filepath.Join(cwd, defaultManifestFilename)
 	if _, err := os.Stat(manifestFilepath); err == nil {
```

Comment:
nit: i know this wasn't part of this PR but should we not check instead that the filename is set on the response? checking for no error to return an error feels weird because we don't check if something else breaks with finding the file. 

If we check for the filename, then we can return an error if it exists as well.

Reply:
if this is the default file checking way, i will accept the method haha. Just wish it didn't look like this

Reply:
Go is such a wonderful language